src/LiveApi.js

```diff
@@ -232,4 +234,29 @@ export default class LiveApi {
             this.bufferedExecutes.push(func);
         }
     }
+
+    // TODO: should we deprecate this? preserve for backward compatibility
+    send = (json: Object): ?LivePromise => {
+        const reqId = getUniqueId();
+        return this.sendRaw({
+            req_id: reqId,
+            ...json,
+        });
+    }
+
+    // TODO: should we deprecate this? preserve for backward compatibility
+    sendRaw = (json: Object): ?LivePromise => {
```

Comment:
sendRaw is definitely needed.
Used by developers.binary.com and possibly some 3rd parties. (or not)